File: sandbox-runner/patterns/continue-as-new/java/src/main/java/Worker.java
import io.temporal.client.WorkflowClient;
import io.temporal.serviceclient.WorkflowServiceStubs;
import io.temporal.worker.WorkerFactory;

public class Worker {
    public static void main(String[] args) {
        WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
        WorkflowClient client = WorkflowClient.newInstance(service);

        WorkerFactory factory = WorkerFactory.newInstance(client);
        io.temporal.worker.Worker worker = factory.newWorker(Shared.TASK_QUEUE);
        worker.registerWorkflowImplementationTypes(DataProcessorWorkflow.Impl.class);
        worker.registerActivitiesImplementations(new Activities.Impl());

        System.out.println("Worker listening on task queue '" + Shared.TASK_QUEUE + "'");
        factory.start();

        try {
            Thread.currentThread().join();
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
        }
    }
}

File: sandbox-runner/patterns/early-return/python/worker.py
import asyncio

from temporalio.client import Client
from temporalio.worker import Worker

from activities import cancel_transaction, complete_transaction, init_transaction
from shared import TASK_QUEUE
from workflows import TransactionWorkflow


async def main() -> None:
    client = await Client.connect("localhost:7233")
    worker = Worker(
        client,
        task_queue=TASK_QUEUE,
        workflows=[TransactionWorkflow],
        activities=[init_transaction, complete_transaction, cancel_transaction],
    )
    print(f"Worker listening on task queue '{TASK_QUEUE}'", flush=True)
    await worker.run()


if __name__ == "__main__":
    asyncio.run(main())
